spec/formatting.md

```diff
@@ -8,6 +8,15 @@ when formatting a message for display in a user interface, or for some later pro
 The document is part of the MessageFormat 2.0 specification,
 the successor to ICU MessageFormat, henceforth called ICU MessageFormat 1.0.
 
+## Literal Resolution
+
+The resolved value of _text_, _literal_ and _nmtoken_ tokens
```

Comment:
This will need to include `unquoted` if/once #364 is accepted.

Reply:
The word "value" seems to be getting used in multiple ways in this paragraph.

The first sentence refers to "the resolved value of _text_, _literal_ and _nmtoken_ tokens"; if resolution is the relation that maps character strings in the language defined by the ABNF onto values, then I understand "value" as being used semantically here. The spec doesn't (yet?) define criteria for membership in this set of semantic values, not in the precise way that the ABNF defines membership in the set of syntactically valid messages.

However, the next sentence refers to an "option value", which I take as being a syntactic concept: the token that appears on the right-hand side of the '=' in the `option` nonterminal into the ABNF.

Defining "value" and "resolution" before these terms are used, and replacing "or option value" with "on the right-hand side of an option", might help clarify things.

Reply:
The intended meaning of "resolved value" here is the value that will ultimately get formatted. So for an unquoted literal `42`, it would be the string `'42'`, while for a quoted literal `|foo\|bar|`, it would be the string `'foo|bar'`. For a variable reference `$foo`, it would be the value of the variable, which could really be anything.

My intent would be to explain this term as a part of the bigger formatting PR I'm now working on.

Renaming "option value" does sound like a good idea.

Reply:
As long as "resolved"/"resolution" are defined in the bigger PR, I'm fine with leaving those terms undefined in this one.